packages/core/src/components/Pagination/index.tsx

```diff
@@ -167,34 +183,68 @@ export class Pagination extends React.Component<Props & WithStylesProps> {
     }
 
     return (
```

Comment:
can you update `<T phrase={'%{pageNumber}'} pageNumber={page} context="Showing the current page number" />` to be `page` on line 162? to match miles feedback on previous pr.

Reply:
@stefhatcher done - pushed with changes